### ISSUE TITLE
[Bug] Autostart fails and window stays open (Win11)

### ISSUE THREAD

**Issue by syd711** [syd711/vpin-studio#437]
The autostart .bat file for the server is executed via registry call, but remains open when executing the server.vbs.
I assume it is related to Win11.

**Comment by syd711** [comment]
Elicend — Today at 6:20 PM
Oups
![image](https://github.com/user-attachments/assets/2b77eb9f-b881-483e-abfb-c7411018cb12)

**Comment by syd711** [comment]
AF555 — 09/01/2024 5:07 PM
If you kill the server and double click this .bat file, does it stay open again? Yes
    Does the Studio Server start (tray icon) then? Yes
    Is the folder shown there for the server.vbs file correct? Yes (C:\vPinball\VPin-Studio)
    Can you also send me the server.vbs file?  Attached.


```
Dim WShell
Set WShell = CreateObject("WScript.Shell")
WShell.CurrentDirectory = "C:\vPinball\VPin-Studio\."
WShell.Run "VPin-Studio-Server.exe", 0
Set WShell = Nothing
```

**Comment by syd711** [comment]
Fixed with 3.5.3
